### ISSUE TITLE
ctrl-backspace doesn't work in insert mode

### ISSUE THREAD

**Issue by darthdeus** [jaredpar/VsVim#1630]
When in insert mode in VsVim, ctrl-backspace no longer deletes a word left to the cursor. Is this something that could be configured with keybindings?

**Comment by jaredpar** [comment]
@darthdeus that should be working.  Can you check in the VsVim key bindings page if `Ctrl-Back` is bound to VsVim or Visual Studio?

**Comment by darthdeus** [comment]
Sorry for replying so late, I completely forgot about this, until I stumbled upon the problem a few days ago again.

I'm not sure if it's VS2015, but I did reset keybindings to default, then re-applied Resharper and then VsVim ones, and ctrl-backspace works. I tried setting the actual hotkey manually (Edit.WordDeleteToStart), but that didn't help without resetting the hotkeys.

I'm not sure if I didn't miss anything, but resetting certainly helped, so at least posting this here for future reference :)
